ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/byo-route53/provision/cucushift-installer-rehearse-aws-ipi-byo-route53-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-byo-route53-provision
+  steps:
+    - ref: aws-provision-vpc-shared
+    - ref: aws-provision-route53-private-hosted-zone
+    - ref: ipi-conf
+    - ref: ipi-conf-aws-custom-vpc
+    - ref: ipi-conf-aws-byo-route53
+    - ref: ipi-conf-aws
+    - ref: ipi-install-monitoringpvc
+    - chain: ipi-install
+  documentation: |-
+    Create an IPI cluster on AWS for QE e2e tests.
```

Comment:
This, and the `Destroy cluster` `documentation` are pretty brief.  Can you add some words about the type of cluster being created/destroyed, and why you feel that type of cluster is useful to handle in CI?